doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -594,6 +594,159 @@ must be sent.
 *default location*, then the query will be declared as ambigous and an HTTP error response with code ``409``
 must be sent. 
 
+## Notifications formats
```

Comment:
A the end, this section talks about an HTTP header (you will see in next commit), so striclty speaking is not only about "payload". I will use "Notificaion messages" at the end.